test/jdk/java/awt/color/ICC_ProfileSetNullDataTest.java

```diff
@@ -30,22 +30,23 @@
  * @summary Test checks behavior of the ICC_Profile.setData(int, byte[])
  */
 public final class ICC_ProfileSetNullDataTest {
+    private static final int[] colorSpace = {
+            ColorSpace.CS_sRGB, ColorSpace.CS_LINEAR_RGB,
+            ColorSpace.CS_PYCC, ColorSpace.CS_GRAY,
+            ColorSpace.CS_CIEXYZ
+    };
```

Comment:
With this modification, we will lose information about which color space is broken from the stack trace. So, at the least, you should print it in case of failure.

Reply:
Updated